### PR TITLE
Docs: monthly reports

### PR DESCRIPTION
**Pull request by xlson**
**What this PR does / why we need it**:

docs for the monthly reports schedule

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/enterprise/reporting.md [hunk 1]
```diff
@@ -80,6 +80,16 @@ Weekly reports are generated once per week. All fields are required.
 * **Time -** Time of day in 24 hours format when the report should be sent.
 * **Time zone -** Time zone for the **Time** field.
 
+#### Monthly
+
+> Only available in Grafana Enterprise v7.1+.
+
+Monthly reports are generated once per month. All fields are required.
+
+* **Day in month -** Day of the month the when report should be sent. (it's possible to select `last` for reports that should go out on the last day of the month).
```
Comment (AgnesToulet) [inline docs/sources/enterprise/reporting.md:89]:
```suggestion
* **Day in month -** Day of the month when the report should be sent (it's possible to select `last` for reports that should go out on the last day of the month).
```
Just noticed that I rewrote all dayInMonth variables to dayOfMonth and forgot to rename the field. I think it should be "Day of month" or "Day of the month"?

**Review by AgnesToulet — COMMENTED** [review]
Great that you thought about this! Looks good except for a small typo and I think we should rename the first field in both the doc and the code.

File: docs/sources/enterprise/reporting.md (hunk 1 above)
Comment (oddlittlebird) [reply docs/sources/enterprise/reporting.md:89]: If it were regular writing, it would be "day of the month", but for variables, shorter seems better.

Comment (oddlittlebird) [reply docs/sources/enterprise/reporting.md:89]: Remove the parenthesis and replace "it's possible to" with "You can".

File: docs/sources/enterprise/reporting.md [hunk 2]
```diff
@@ -80,6 +80,16 @@ Weekly reports are generated once per week. All fields are required.
...
+* **Time -** Time of day in 24 hours format when the report should be sent.
```
Comment (oddlittlebird) [inline docs/sources/enterprise/reporting.md:90]:
```suggestion
* **Time -** Time the report is sent, in 24-hour format.
```

**Review by oddlittlebird — DISMISSED** [review]
A few suggestions.

File: docs/sources/enterprise/reporting.md (hunk 1 above)
Comment (xlson) [reply docs/sources/enterprise/reporting.md:89]: Thank you, I will incorporate the feeback!

File: docs/sources/enterprise/reporting.md (hunk 2 above)
Comment (xlson) [reply docs/sources/enterprise/reporting.md:90]: Thank you, I will incorporate the feeback!

**Review by AgnesToulet — APPROVED** [review]
LGTM!

**Pull request merged**